src/views/Home/AlertsView/index.tsx

```diff
@@ -43,10 +52,17 @@ query CountryList($alertFilters: AlertFilter) {
 `;
 
 const COUNTRY_ALERTS_COUNT = gql`
-query CountryAlertsCount {
-    public {
-      alerts {
+query CountryAlertsCount ($filters: AlertFilter){
+    public{
+      alerts(filters: $filters) {
         count
+        items {
+            country {
+                id
+                name
+                alertCount
+            }
+        }
```

Comment:
Let's add `id` to the `public` node. Apollo graphql uses `normalized` caching. Please go through this https://www.apollographql.com/docs/react/caching/overview/ on how it is done and why the cache id is required.  @barshathakuri @roshni73